yeelight/flow.py

```diff
@@ -44,6 +44,10 @@ def __init__(self, count=0, action=Action.recover, transitions=None):
         self.action = action
         self.transitions = transitions
 
+        # Note, main depends on us, so we cannot import BulbException here.
+        if len(self.transitions) > 10:
+            raise Exception("More transactions (%s) than maximum of 10." % len(self.transitions))
```

Comment:
Yes, I think a WARNING would be ideal for now. Also, it should be `> 9`, as that's how many gave me problems.